frontend/src/static/js/v_summary.js

```diff
@@ -532,6 +532,7 @@ window.vSummary = {
       });
 
       this.checkedFileTypes = this.fileTypes.slice();
+      this.$store.commit('updateFileTypeColors', this.fileTypeColors);
```

Comment:
Is this line related to this PR?

Reply:
Oh sorry, that somehow got copied into this branch. I'll take it out.